Send section

Sirmorrison opened the pull request:
### Resolves

Issue #

### What's new

This PR uses card to create sections on the send page to look like the mobile app.

### Relevant screenshots or logs
<img width="799" alt="image" src="https://user-images.githubusercontent.com/27733432/88285638-5ee81080-cce7-11ea-8c13-4540ad708d8f.png">

<img width="794" alt="image" src="https://user-images.githubusercontent.com/27733432/88369569-349a5f80-cd88-11ea-9cb6-5f7879144d0e.png">

<img width="798" alt="image" src="https://user-images.githubusercontent.com/27733432/88369525-1f253580-cd88-11ea-9b93-92c7e9fd1eda.png">

Sirmorrison committed:
use cards to make sections on send page
Add max wallet spendable balance option
Add spendible balace to amount section

Sirmorrison committed:
add outline to send page function
add send to account option
add is outline data type to editor widget

Sirmorrison committed:
add collapasible and line widget for txfee
move selected account display card to page.go
move txfee to accountinputsection just as mobile layout

Sirmorrison committed:
rebase and fix broken pages after rebase

Sirmorrison committed:
add section seperator lines

Sirmorrison committed:
use decredmaterail outline widget

Sirmorrison committed:
use decredmaterial outline widget for sections

Sirmorrison committed:
use gioui border widget instead of decredmaterial outline
use border for sync button on overview page

Sirmorrison committed:
add line between page content and account tab on page scroll

Sirmorrison committed:
remove left and right tab padding from layout
remove uniform padding from page content\nextend separator line to cover page width
add left padding to tab widget
adjust top padding of tab items to overview button hieght

oshorefueled commented on ui/page.go:
```diff
@@ -135,8 +136,8 @@ func (page pageCommon) Layout(gtx layout.Context, body layout.Widget) layout.Dim
-		p := values.MarginPadding10
-		return layout.Inset{Top: p, Left: p, Right: p}.Layout(gtx, func(gtx C) D {
+		p := values.MarginPadding5
+		return layout.Inset{Bottom: p}.Layout(gtx, func(gtx C) D {
```
The padding is used just once, it can be used directly in the inset. There's no need to initialize it first.

Sirmorrison committed:
fix issue with padding when tab aligns to left or right

Sirmorrison committed:
rebase pr

oshorefueled commented on ui/values/dimensions.go:
```diff
@@ -28,6 +28,9 @@ var (
 	WalletSyncBoxContentWidth = unit.Dp(280)
 	AccountLineWidth          = unit.Dp(350)
+	OutLinePadding            = unit.Dp(1)
+	BorderWidth               = unit.Px(2)
```
The paddings and margins should follow the normal padding convention. Example: "MarginPadding5". For the negative values, you can use underscore. So, "MarginPadding_5".

oshorefueled commented on ui/send_page.go:
```diff
@@ -312,11 +346,30 @@ func (pg *SendPage) Layout(gtx layout.Context, common pageCommon) layout.Dimensi
+				return layout.Dimensions{}
+			}),
+			layout.Rigid(func(gtx C) D {
+				// return layout.UniformInset(values.MarginPadding10).Layout(gtx, func(gtx C) D {
```
should this be left here?

Sirmorrison committed:
remove commented code
remave dimension values to align with normal convention
align destination address field button

Sirmorrison committed:
fix lint

Sirmorrison committed:
remove commented code
Rename dimension values
